server/db/events.py

```diff
@@ -1,8 +1,8 @@
 """PURPOSE: https://fastapi.tiangolo.com/advanced/events/
 
 TODO: https://docs.sqlalchemy.org/en/14/core/connections.html
-    * Investigate whether there are any significant advantages with managing connections ourselves.
-    * Alternative: use `get_db` with Dependency hook for every API request.
+    * Investigate any advantages managing connections ourselves.
+    * Alternative: `get_db` with Dependency hook for every request.
```

Comment:
We still have to do these lol

Reply:
Yeah, we do. It's just not in scope of this particular ticket. #8 was allocated for this task.

I just refined the comments since the linting process I added enforces line length of 88 (and doc length of 72).